pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -261,6 +261,59 @@ func (a *Actuator) Delete(ctx context.Context, cluster *clusterv1.Cluster, machi
 	return nil
 }
 
+// immutableSateChanged checks that no immutable fields have been updated in an
```

Comment:
I'm wondering if this is the best place for this function to live in. We should either have this in a different package or maybe it could be fine to have it as a machine actuator internal method. A possible name could be `isMachineOutdated`.

Reply:
I had been wondering about the location of a few of these utility type functions, but since there wasn't currently a good looking alternative, I put it next to the method using it. Function has been renamed and is now a method on actuator.